api/test/integration/conftest.py

```diff
@@ -70,8 +70,9 @@ def build_and_up(interval: int = 10, interval_build_env: int = 10, build: bool =
     dict
         Dict with healthchecks parameters.
     """
-    pwd = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'env')
-    os.chdir(pwd)
+    current_abspath = os.path.abspath(os.path.dirname(__file__))
+    env_dir_abspath = os.path.join(current_abspath, 'env')
```

Comment:
Can't these variables be defined at a module level instead of locally since they are used on several functions?

Reply:
Yes, and in fact, they are already defined:

```
current_abspath = os.path.abspath(os.path.dirname(__file__))
```

is the same as

```
current_path = os.path.dirname(os.path.abspath(__file__))
```